src/pages/Checkout/CheckoutPage.vue

```diff
@@ -1003,7 +1019,11 @@ export default {
 							);
 							// eslint-disable-next-line max-len
 							this.$showTipMsg('Looks like that loan was reserved by someone else! Try this one instead.', 'info');
-							this.getUpsellModuleData();
+							if (this.$route.query.forceDynamicUpsell === 'true' && this.enableDynamicUpsells) {
```

Comment:
What's the use case for this query param? Is it temporary?

Reply:
yes really just for testing with product/design! since the dynamic upsell loan condition will only be met if there are no loans with <$50 left.

Reply:
Ok, that makes sense. Just wanted to better understand if this hard condition to have the query param would end up in production and be a dependency there too.